.env

```diff
@@ -3,3 +3,4 @@ VUE_APP_JITSI_URL=meet.aepps.com
 VUE_APP_BACKEND_URL=https://raendom-backend.z52da5wt.xyz
 VUE_APP_CONTRACT_V1_ADDRESS=ct_2AfnEfCSZCTEkxL5Yoi4Yfq6fF7YapHRaFKDJK3THMXMBspp5z
 VUE_APP_CONTRACT_V2_ADDRESS=
+BASE_URL=/
```

Comment:
Won't it be defined by Vue cli? When I was adding SSR to the example aepp, I didn't need to do that: https://github.com/davidyuk/vue-reproductions/commit/1474d4ed25d9d0c452e2af9a5c22d7e1b3444c98

Reply:
@davidyuk cool, so you figure out with that? I will try my best at Monday!

Reply:
@davidyuk thanks, fixed, messed up because of health